sources/platform/actors/running/tasks.md

```diff
@@ -1,51 +1,45 @@
 
-You can create tasks both for your personal Actors and for those made by someone else. Just [search Apify Store](https://console.apify.com/store) to find an Actor you want to use and click the **Create new task** button.
-
-Like any other resource, you can share your Actor tasks with other Apify users via the [access rights](../../collaboration/index.md) system.
+You can find all your tasks at [https://console.apify.com/actors/tasks](https://console.apify.com/actors/tasks).
 
 ## Create
 
-Open any Actor from [Apify Store](https://console.apify.com/store) or your list of [Actors](https://console.apify.com/actors) in Apify Console. At the top-right section of the page, click the **Create empty task** button.
+To create a task, open any Actor from [Apify Store](https://console.apify.com/store) or your list of [Actors](https://console.apify.com/actors) in Apify Console. At the top-right section of the page, click the **Create task** button.
 
 ![Create a new Apify task](./images/tasks/create-task.png)
 
 ## Configure
 
-You can set up your task's input under the **Input and options** tab. A task's input configuration works just like an Actor's. After all, it is just a copy of an Actor you can pre-configure for a specific scenario. You can use either JSON or the visual input UI.
+You can set up your task's input under the **Input and options** tab. A task's input configuration works just like an Actor's. After all, it's just a copy of an Actor you can pre-configure for a specific scenario. You can use either JSON or the visual input UI.
 
 ![Apify task configuration](./images/tasks/create-task-configure.png)
 
-Actors' input fields will vary depending on their purpose, but they all follow the same principle: you provide an Actor with the information it needs so it can do what you want it to do.
+An Actors' input fields may vary depending on their purpose, but they all follow the same principle: _you provide an Actor with the information it needs so it can do what you want it to do._
 
 You can set run options such as timeout and [memory](./usage_and_resources.md) at the bottom of the input section.
 
-### Identifying tasks
+### Identification
 
-To make a task easier to identify, you can give it a name, title, and description by clicking it's caption on the detail page.  A task's name should be 3-63 characters long.
+To make a task easier to identify, you can give it a name, title, and description by clicking its caption on the detail page. A task's name should be at least 3 characters long with 63 characters limit.
 
 ### Settings
 
-You can grant [access rights](../../collaboration/index.md) to other Apify users under the **Settings** tab of their detail page.
+You can grant [access rights](../../collaboration/index.md) to other Apify users under the **Settings** tab of the task's detail page.
```

Comment:
Sharing is now under this button

![CleanShot 2024-03-21 at 11 33 55@2x](https://github.com/apify/apify-docs/assets/38246758/ec8e96dc-2c7c-4606-9109-6ea22d5051b2)